### PR TITLE
Remove unneeded `createProperties` object to display compat data

### LINKED ISSUE #20969: webextensions.api.tabs.create - Chromium doesn't support `discarded` (throws an error when used)
### What type of issue is this?

Missing compatibility data

### What information was incorrect, unhelpful, or incomplete?

tabs.create in Chromium browsers does not support discarded property. It throws out an error when it is used. 

As someone who uses MDN compatibility data to develop cross-browser extensions, it was a little bit hard to see where the problem was when using the same API with the same properties and figuring out it stops the extension from working. 

### What browsers does this problem apply to, if applicable?

Chromium (Chrome, Edge 79+, Opera, Samsung Internet)

### What did you expect to see?

I expect to add a notice to indicate the missing properties support or at least include the supported properties to the table.

For example: In Chromium browsers, tabs.create does not support the `discarded` property.

And even better, indicate that Firefox supports Firefox only properties such as `openInReaderMode` and `cookieStoreId`.

### Did you test this? If so, how?

I figured this out while creating my extension [Sessionic](https://github.com/navorite/sessionic), so I went back to the documentation multiple times only to figure out on the Chrome documentation that they do not support `discarded`.

### Can you link to any release notes, bugs, pull requests, or MDN pages related to this?

https://developer.chrome.com/docs/extensions/reference/tabs/#method-create

### Do you have anything more you want to share?

_No response_

### MDN URL

https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/tabs/create

### MDN metadata

<details>
<summary>MDN page report details</summary>

* Query: `webextensions.api.tabs.create`
* Report started: 2023-10-17T02:51:22.810Z

</details>

### PR DESCRIPTION
**Pull request by navorite**
#### Summary

This removes the nesting of `createProperties` object parameters to display the compatibility data documentation in the `tabs.create` API.

Fixes #20969

### CODE REVIEW

**Review by Bobain95 — APPROVED** [review]

File: webextensions/api/tabs.json [hunk 1]
```diff
@@ -1486,274 +1486,272 @@
...
-                  }
+          "active": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "5"
+                },
+                "edge": {
+                  "version_added": "14"
+                },
```
Comment (rebloor) [inline webextensions/api/tabs.json:1497]: Please amend to show Edge as "mirror"

File: webextensions/api/tabs.json [hunk 2]
```diff
@@ -1486,274 +1486,272 @@
...
+          },
+          "index": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "5"
+                },
+                "edge": {
+                  "version_added": "14"
+                },
```
Comment (rebloor) [inline webextensions/api/tabs.json:1565]: Please amend to show Edge as "mirror"

File: webextensions/api/tabs.json [hunk 3]
```diff
@@ -1486,274 +1486,272 @@
...
+          "url": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "5"
+                },
+                "edge": {
+                  "notes": "If the <code>url</code> has the 'ms-browser-extension://' protocol it is mistakenly considered a relative URL and the prefix is added redundantly, causing tab to fail loading.",
+                  "version_added": "14"
+                },
```
Comment (rebloor) [inline webextensions/api/tabs.json:1715]: Please amend to show Edge as "mirror"

File: webextensions/api/tabs.json [hunk 4]
```diff
@@ -1486,274 +1486,272 @@
...
+          },
+          "windowId": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "5"
+                },
+                "edge": {
+                  "version_added": "14"
+                },
```
Comment (rebloor) [inline webextensions/api/tabs.json:1742]: Please amend to show Edge as "mirror"

**Review by rebloor — CHANGES_REQUESTED** [review]

**Review by rebloor — APPROVED** [review]

**Pull request merged**